src/eva/vision/utils/io/image.py

```diff
@@ -52,3 +52,30 @@ def read_image_as_array(path: str, flags: int = cv2.IMREAD_UNCHANGED) -> npt.NDA
         image = image[:, :, np.newaxis]
 
     return np.asarray(image).astype(np.uint8)
+
+
+def get_mask(
```

Comment:
I'm not sure if this belongs here, as it's not really IO?
Maybe in `src/eva/vision/data/wsi/masking/kaggle.py` (replace kaggle by a better name)

Reply:
how about a `mask.py` in `src/eva/vision/utils`? I'm thinking eventually we'll only have one get_mask function (if we pick what works best, there should be no need for alternatives)

Reply:
`src/eva/vision/utils/mask.py` I think would be fine for now (I'd move it out of coordinates). We can later move / refactor this part when we want to support multiple masking methods.